File: eva-core/eva-core-data-mybatis/src/main/java/io/nerv/core/exception/mapper/ErrorlogMapper.java
package io.nerv.core.exception.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import io.nerv.core.exception.entity.ErrorlogEntity;
import org.apache.ibatis.annotations.Mapper;
import org.springframework.stereotype.Repository;

/**
 * @author: S.PKAQ
 * @Datetime: 2018/9/27 8:40
 */
@Mapper
@Repository
public interface ErrorlogMapper extends BaseMapper<ErrorlogEntity> {
}

File: eva-core/src/main/kotlin/io/nerv/exception/mapper/ErrorlogMapper.kt
package io.nerv.exception.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import io.nerv.exception.entity.ErrorlogEntity
import org.apache.ibatis.annotations.Mapper
import org.springframework.stereotype.Repository

/**
 * @author: S.PKAQ
 * @Datetime: 2018/9/27 8:40
 */
@Mapper
@Repository
interface ErrorlogMapper : BaseMapper<ErrorlogEntity?> 